apps/core/scenes/resultDetail/ResultDetail.js

```diff
@@ -25,9 +28,12 @@ export default class ResultDetail extends React.Component<Props> {
         `}
         variables={{
           input: {
-            bookingToken: this.props.bookingToken,
+            bookingToken,
             bags: 0, // @TODO - use real data from search context
-            passengers: { adults: 1 }, // @TODO - use real data
+            passengers: {
+              adults: adults ? parseInt(adults, 10) : 1,
```

Comment:
I'm not that happy about this part. Mainly because we are mixing values from url and search context in few places. I want to do some refactoring in different PR and use SearchContext as main source and parser of parameters for Search/Results/ResultDetail . I just need to sync with @aalises  because he will probably need something similar for search input above web results.

Reply:
I guess this is string | number because it is passed differently from web and mobile?

How about doing the parseInt on the parent, so the type here can be just number?

Reply:
@tbergq Good idea. Moved parsing to parent for web version (where params are string) which makes it a bit cleaner. Thanks